src/css.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.module.css" {
+  const classes: { [key: string]: string };
+  export default classes;
+}
```

Comment:
pick a lane :p

Reply:
It's not quite working anyway :yum:

That's how all the examples online do it, but it's throwing an error.